### PR TITLE
Update README re adding AWS Account IDs

### PR DESCRIPTION
**Pull request by avnerenv0**
closes https://github.com/gruntwork-io/terragrunt-infrastructure-live-example/issues/50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ you are interested in battle-tested, production-ready Terraform code, check out
    the file is already taken, so you'll have to specify your own. Alternatives, you can
    set the environment variable `TG_BUCKET_PREFIX` to set a custom prefix.
 1. Configure your AWS credentials using one of the supported [authentication
-   mechanisms](https://www.terraform.io/docs/providers/aws/#authentication).
+   mechanisms](https://www.terraform.io/docs/providers/aws/#authentication), and fill in your AWS Account ID's in `prod/account.hcl` and `non-prod/account.hcl`.
```
Comment (brikis98) [inline README.md:37]: Probably better to call this out on a separate line:

```suggestion
   mechanisms](https://www.terraform.io/docs/providers/aws/#authentication).
1. Fill in your AWS Account ID's in `prod/account.hcl` and `non-prod/account.hcl`.
```

**Review by brikis98 — COMMENTED** [review]
Thanks for the PR!

**Comment by avnerenv0** [comment]
@brikis98 done

**Review by brikis98 — APPROVED** [review]
Thank you!

**Pull request merged**